app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -437,6 +438,13 @@ object ComponentRegistry {
         TAB_TITLE_VIEW story NodeData { Story of R.layout.component_list },
       )
     ),
+    "Skeleton" story NodeData(
+      { children -> TabStory of children },
+      mapOf(
+        TAB_TITLE_COMPOSE composeStory { SkeletonStory() },
```

Comment:
nitpick: we generally put the view tab first

Reply:
ok.